projects/HotFire/config/config.h

```diff
@@ -1,15 +1,15 @@
 #pragma once
 
 #ifndef USE_SBG
-    #define USE_SBG 1
+    #define USE_SBG 0
 #endif
 
 #ifndef USE_SOCKET_CLIENT
     #define USE_SOCKET_CLIENT 0
 #endif
 
 #ifndef USE_LOGGER
-    #define USE_LOGGER 1
+    #define USE_LOGGER 0
```

Comment:
No logs?

Reply:
Oops, changed this for testing. I will put it back to 1